bot.py

```diff
@@ -12,6 +14,30 @@
 # target_channel_id = 722609125950750771 # testing
 activity = True
 
+@bot.event
+async def on_message(message):
+    if message.author == client.user:
```

Comment:
what does this do?

Reply:
prevents the bot from retriggering - it ignores messages that it itself sent

Reply:
in that case you can make it `if not message.author.bot:` and move everything else under it